crates/dotrain/src/composer/mod.rs

```diff
@@ -1115,6 +1115,21 @@ _: opcode-1(0xabcd "something.else");
 _: opcode-1(0xabcd "something.else");"#;
         assert_eq!(rainlang_text, expected_rainlang);
 
+        let dotrain_text = r#"
+        some 
+        front 
+        matter
+---
+/* this is test */
+
+#exp-binding
+_: opcode-1(12.34e6 123.123);
```

Comment:
https://github.com/rainlanguage/rain.interpreter/blob/main/test/src/lib/parse/literal/LibParseLiteralDecimal.parseDecimal.t.sol 

here is an example of how many tests we should have, i want 10-100x more test cases

Reply:
added fuzz tests for num literals